portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,20 +14,35 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  private List<String> messages = new ArrayList<>(Arrays.asList("Brilliant!", "Pretty!", "Great!"));
```

Comment:
Do we need the new ArrayList<> ? or just Arrays.asList(a, b, c) is sufficient?
Actually, since we're not modifying these messages anywhere. Maybe enforce it with
private static final ImmutableList<String> messages = ImmutableList.of(a, b, c);
It's easier to deal with immutable data.

Reply:
This one is the hard coded one, will need modification in the later tutorial. so i will just modify to "Arrays.asList(a,b,c)"

Reply:
Fixed